swarm/runtime/evolution_processor.py

```
@@ -0,0 +1,741 @@
+                    flow_key="wisdom",
+                    payload={
+                        "patch_id": patch.id,
+                        "target_file": patch.target_file,
+                        "reason": "; ".join(apply_result.errors),
+                        "policy": policy.value,
+                    },
+                )
+
+            result.suggestions.append(suggestion)
+
+        except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)